api/static_route.go

```diff
@@ -0,0 +1,13 @@
+package api
+
+import (
+	"github.com/canonical/gomaasclient/entity"
+)
+
+// StaticRoute is an interface defining API behaviour for
```

Comment:
nit: Since we changes to US English. I know that the code contains `behaviour` in many places but let's try not to introduce new occurrences.

```suggestion
// StaticRoute is an interface defining API behavior for
```